go-controller/pkg/node/gateway_localnet.go

```diff
@@ -306,57 +254,18 @@ func (l *localPortWatcher) deleteService(svc *kapi.Service) error {
 }
 
 func (l *localPortWatcher) SyncServices(serviceInterface []interface{}) {
```

Comment:
on sync services you need to remove all routes for table 6 so that on upgrade to this new method of using iptables, the legacy routes are removed.

Reply:
Ack done